bridge/MixerManager.cpp

```diff
@@ -276,6 +276,16 @@ void MixerManager::stop()
             break;
     }
 
+    bool jobsDone = false;
```

Comment:
dead lock at test shutdown. I believe it can happen in production too if there is a race for removing Mixer object when Bridge is in ~Bridge.